src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * 40) - 20;
+  }
+
+  if (days >= 7) {
+    return (days * 40) - 50;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return days * 40;
  }

  if (days >= 3 && days < 7) {
    return (days * 40) - 20;
  }
  
  return (days * 40) - 50;
 
```

Reply:
also move 20, 40 and 50 in the variables

Reply:
Hi, Tatiana.
Ok, understood and corrected the error)

Reply:
Hi. Please check. If everything is ok, then confirm)